src/page-components/Home/Notice/index.tsx

```diff
@@ -37,9 +38,10 @@ function Heterogeneous() {
     [isHomogeneous, toChainId],
   );
   const time = useMemo(() => getCrossChainTime(fromChainId, toChainId), [toChainId, fromChainId]);
-  return (
-    <>
-      {isHeterogeneousCrossInChain ? (
+
+  const getContent = () => {
```

Comment:
There is no need to declare a separate arrow function, you can just return the contents of this function.

Reply:
Updated